### DISCUSSION TITLE
Office UI Fabric vs. Fluent UI > is the documentation still valid?

### DISCUSSION THREAD

**Discussion by chr-sad** [SharePoint/sp-dev-docs#8361] [Q&A] [answered]
We have a SPFx solution (1.15.2) containing WebParts currently based on Office UI Fabric React 7.x. According to the documentation I found here, we should still be using Office UI Fabric instead of Fluent UI:
https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/use-fabric-react-components

I was wondering, if this is still valid?

Would it be supported to use Fluent UI React 8.x in the latest version in our SPFx solution in an productive environment as well because they have many features / improvements we would like to take use of or what is the reason to don't use this version right now?

Thank you :)

**Comment by chr-sad** [comment]
Does anyone know something about this topic? Currently there is already version 8.98.3 released with improvements compared to 7.x so what is the reason to not use 8.x versions of Fluent UI?

**Comment by AJIXuMuK** [comment, marked as answer]
@chr-sad - SPFx is still based on OUIFR v7. You can use Fluent UI package, but still v7.
You can try to use v8 but we can't guarantee it is working properly.

**Comment by chr-sad** [comment]
@AJIXuMuK  thank's for clarification. Do you know if there is a timeline for v8 support or if there are any plans to upgrade this dependency?

**Reply by AJIXuMuK** [reply]
There are plans, relatively soon :)

**Reply by chr-sad** [reply]
Thanks :)

**Comment by chr-sad** [comment]
@AJIXuMuK we recently updated to SPFx 1.17.0 but if I am correct and according to the documentation we should still use OUIFR v7 right? Do you already know in which SPFx version this change to v8.x is planned? The problem with v7 is some missing features which would be already available in v8 (like tab overflow menu)

**Reply by AJIXuMuK** [reply]
@chr-sad - that is correct, SPFx 1.17 is still based on OUIFR v7.
We're planning to update to v8 relatively soon.
